### PR TITLE
Added Godzilla vs. Mechagodzilla (1974) 

### PR DESCRIPTION
**Pull request by bluewhaledelta**
Updated with the movie "Godzilla vs. Mechagodzilla" (1974).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1419,6 +1419,14 @@ Categories: `apocalypse` `aliens` `action`
 
 > American nuclear weapons testing results in the creation of a seemingly unstoppable, dinosaur-like beast.
 
+#### [Godzilla vs. Mechagodzilla (1974)](https://www.imdb.com/title/tt0071565/) (1974) [6.4]
+
+Categories: `action` `fantasy` `horror`
+
+> Ape-like aliens build a robotic Godzilla to destroy Japan, and the true Godzilla may not be powerful enough to destroy it. 
+
+Aliens attack Godzilla with a metallic clone of himself. The film features an adventurous spy sub-plot on a cruise from Japan to Okinawa, giant robots, mystical monsters and radioactive beasts -- and the film is not without its charm. - [@bluewhaledelta] (https://github.com/bluewhaledelta)
```
Comment (RichardLitt) [inline readme.md]: Move this above the description?

Comment (RichardLitt) [reply readme.md]: This was not resolved.

Comment (RichardLitt) [reply readme.md]: Also - why do you think this is an awesome film?

Comment (bluewhaledelta) [reply readme.md]: Ok, yes, line 1428 can be moved to be above the description.

The film (in my opinion) was good as it was the first appearance of Mechagodzilla, and had all the Godzilla hallmarks: giant monsters, explosions, fights, and destruction - also, the dramatic scenes were more fun as they were of the spy/adventure variety.

Comment (RichardLitt) [reply readme.md]: Awesome! Do you need help learning how to edit a PR so you can make these changes, so that this can be merged?

Comment (bluewhaledelta) [reply readme.md]: Oh! Yes, I would like to learn how to do that.

Comment (RichardLitt) [reply readme.md]: Option 1 should help! https://www.burntfen.com/2015-10-30/how-to-amend-a-commit-on-a-github-pull-request

**Comment by sindresorhus** [comment]
@bluewhaledelta Ping

**Pull request closed without merging**